VirtualReality/Program.cs

```diff
@@ -1,29 +1,155 @@
-﻿using System;
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
 using System.Net.Sockets;
 using System.Threading;
 
 namespace VirtualReality
 {
     class Program
     {
-        
+
+
+        private NetworkStream networkStream;
+        private Dictionary<string, string> userSessionsMap;
 
         static void Main(string[] args)
         {
+            Program program = new Program();
+            program.Start();
+        }
+
+        public Program()
+        {
+            // Initialise and connect to the TcpClient
+            // On server: 145.48.6.10 and port: 6666
             TcpClient client = new TcpClient();
             client.Connect("145.48.6.10", 6666);
-            string data = @"{""id"" : ""session/list""}";
-            byte[] byteData = System.Text.Encoding.ASCII.GetBytes(data);
-            int length = byteData.Length;
+            // Request the session list from the server
+            networkStream = client.GetStream();
+            userSessionsMap = GetRunningSessions();
+        }
+
+        /// <summary>Start does <c>The beginning of the Program</c> This is the beginning of the program, als sometimes called the start of a programs life</summary>
```

Comment:
```suggestion
        /// <summary>Start does <c>The beginning of the Program</c> This is the beginning of the program, als 
        /// sometimes called the start of a programs life</summary>
        ///
        
```

Witregels onder comment volgens style guide.